.github/files/gh-autotagger/workflows/autotagger.yml

```
@@ -107,3 +107,57 @@ jobs:
             git tag --force "${TAG%%.*}"
             git push --force origin "${TAG%%.*}"
           fi
+  tag-rolling:
+    name: Rolling mode tag
+    runs-on: ubuntu-latest
```

(this goes with line 136 below)
```suggestion
    runs-on: ubuntu-latest
    if: github.ref == 'refs/heads/trunk'
```